docs/guide.adoc

```diff
@@ -63,9 +64,9 @@ In this example we see two functions. First, we define a new function and store
 it in the constant variable `greet`. We also use the print-function in the
 function body and print a string to the terminal.
 
-The function block starts with `do` and ends with `end`. The `do` can be
-replaced with a `->` if the function returns something. See the following
-examples.
+The function block starts with `do` or `->` and ends with `end`. `do` indicates
+no return value while `->` means the function returns something. See the
+following examples.
```

Comment:
`do` doesn't require no return value. Although I can't seem to think of a good rewording.

Reply:
I think the whole sentence has to be restructured